README.md

```diff
@@ -0,0 +1,6 @@
+# Captcha.EU 
+Typo3 extension for captcha.eu see <a href="https://docs.captcha.eu/typo3-install?id=-form-beta">Documentation</a>

```

Comment:
please change `Typo3` to `TYPO3` - would also be great if you could adopt that at your documentation - some guys and girls are a bit picky on that topic and the trademark is also all uppercase